contract/src/lib.rs

```diff
@@ -70,36 +73,53 @@ impl Default for VersionedMpcContract {
     }
 }
 
+#[derive(BorshDeserialize, BorshSerialize, Serialize, Deserialize, Debug, Clone)]
+pub struct SignatureRequest {
```

Comment:
Why do we need SignatureRequest and SignRequest? Can we use 1 struct?

Reply:
SignatureRequest describes all the information about a request required by the signing protocol, SigRequest is the structure of the request sent to the contract, so they're different. That being said, we can probably come up with better names.